src/tests/backend/specs/api/pad.js

```diff
@@ -586,6 +588,25 @@ describe(__filename, function () {
     });
   });
 
+  describe('appendText', function () {
+    it('Append text to a pad Id', async function () {
```

Comment:
This is inside a different `description()`, so it might be OK that it shares the same name as another `it()`. But it might be a good idea to change it anyway.

Reply:
Note that if we apply https://github.com/ether/etherpad-lite/pull/5190#discussion_r716116155 then this is mute.